cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -145,8 +145,8 @@ private void addStepsFromTable(PhaseTapChangerAdder ptca) {
         Comparator<PropertyBag> byStep = Comparator.comparingInt((PropertyBag p) -> p.asInt("step"));
         table.sort(byStep);
         for (PropertyBag point : table) {
-            double alpha = point.asDouble("angle");
-            double rho = point.asDouble("ratio", 1.0);
+            double alpha = -point.asDouble("angle");
+            double rho = 1 / point.asDouble("ratio", 1.0);
```

Comment:
Could you add a comment here and below to explain which kind of inversion you are doing. Regarding to this function, it seems you change from CGMES convention to IIDM convention, and the inversion done below is relative to the PTC position.

Reply:
Done